EIPS/eip-draft_request_id.md

```diff
@@ -0,0 +1,155 @@
+
+## Simple Summary
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+This document proposes a way to slightly increase the efficiency of the `eth` networking protocol while at the same time reducing complexity in Ethereum node implementations.
+
+## Abstract
+<!--A short (~200 word) description of the technical issue being addressed.-->
+The `eth` protocol defines various request and response commands that are used to exchange data between Ethereum nodes. For example, to ask a peer node for a specific set of headers, a node sends it the [`GetBlockHeaders`](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#getblockheaders-0x03) command.
+
+*Citing from the [`GetBlockHeaders` spec definition](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#getblockheaders-0x03):*
+
+>`[block: {P, B_32}, maxHeaders: P, skip: P, reverse: P in {0, 1}]`
+
+>Require peer to return a `BlockHeaders` message. Reply must contain a number of block
+headers, of rising number when `reverse` is `0`, falling when `1`, `skip` blocks apart,
+beginning at block `block` (denoted by either number or hash) in the canonical chain, and
+with at most `maxHeaders` items.
+
+The node that receives the `GetBlockHeaders` command should answer it with the [`BlockHeaders`](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#blockheaders-0x04) response command accordingly.
+
+*Citing from the [`BlockHeaders` spec definition](https://github.com/ethereum/devp2p/blob/master/caps/eth.md#blockheaders-0x04):* 
+
+>`[blockHeader_0, blockHeader_1, ...]`
+
+>Reply to `GetBlockHeaders`. The items in the list (following the message ID) are block
+headers in the format described in the main Ethereum specification, previously asked for
+in a GetBlockHeaders message. This may validly contain no block headers if none of the
+requested block headers were found. The number of headers that can be requested in a
+single message may be subject to implementation-defined limits.
+
+Let's consider a client making many simultanous requests for `GetBlockHeaders` to one of its peers. By nature it can not be guaranteed that the expected responses arrive in the same order as they were sent. For the client to associate the incoming responses to the correct requests it has to loop through all pending requests trying to match it with the incoming response based on its contents.
+
+This can be particular tricky for responses that are ambigious such as empty responses.
+
+Now consider we change both the `GetBlockHeaders` and the `BlockHeaders` command to include a `request_id`.
```

Comment:
This would be a little more clear if instead of "now consider we change" if had "this EIP changes". This is line 48 and the EIP still hasn't concretely said what it's proposing except in the title!